src/qml/bitcoin.cpp

```diff
@@ -292,6 +294,8 @@ int QmlGuiMain(int argc, char* argv[])
     qmlRegisterSingletonInstance<AppMode>("org.bitcoincore.qt", 1, 0, "AppMode", &app_mode);
     qmlRegisterType<BlockClockDial>("org.bitcoincore.qt", 1, 0, "BlockClockDial");
     qmlRegisterType<LineGraph>("org.bitcoincore.qt", 1, 0, "LineGraph");
+    qmlRegisterUncreatableType<PeerDetailsModel>("org.bitcoincore.qt", 1, 0, "PeerDetailsModel", "");
```

Comment:
for reviewers: https://doc.qt.io/qt-6/qqmlengine.html#qmlRegisterUncreatableType